GSA/made-in-america
# [MIA-310] Update Cypress to version 9.5.2

## Description

link to ticket: https://cm-jira.usa.gov/secure/RapidBoard.jspa?rapidView=3372&projectKey=FORMS&view=detail&selectedIssue=MIA-310

Cypress released a patch update, in order to keep our tech up to date and have the best support we need to upgrade Cypress to version 9.5.2.  This update included some bug fixes from Cypress, the link to the changelog is below. 

https://docs.cypress.io/guides/references/changelog#9-5-2

## How to test

- Checkout branch `MIA-310`
- Pull down the latest 
- `npm install`
- `npm run start`
- spilt terminal 
- `npm run test` will run all Cypress test headless 
- Ensure tests are passing 

## Additional information

This PR will help keep Cypress up to date and avoid any technical debt 

To prepare for code review, please follow the following checks.
